packages/components/psammead-social-embed/src/Canonical/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { memo } from 'react';
+import Helmet from 'react-helmet';
+import { shape, string } from 'prop-types';
+import loadable from '@loadable/component';
+
+const OEmbed = loadable(() => import('@bbc/psammead-oembed'), {
+  fallback: <p>Loading&hellip;</p>,
+});
+
+export const providers = {
+  instagram: {
+    script: 'https://www.instagram.com/embed.js',
+  },
+  twitter: {
+    script: 'https://platform.twitter.com/widgets.js',
+  },
+  youtube: {},
```

Comment:
Might be worth adding a comment to just state that a JS widget for youtube is not required.